presto-mongodb/src/test/java/io/prestosql/plugin/mongodb/MongoServer.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.mongodb;
+
+import org.testcontainers.containers.GenericContainer;
+
+import java.io.Closeable;
+
+public class MongoServer
+        implements Closeable
+{
+    private static final int MONGO_PORT = 27017;
+
+    private final GenericContainer<?> dockerContainer = new GenericContainer<>("mongo:3.2.0");
+
+    public MongoServer()
+    {
+        dockerContainer.withExposedPorts(MONGO_PORT)
+                .withEnv("MONGO_INITDB_DATABASE", "tpch")
+                .withCommand("--bind_ip 0.0.0.0")
+                .start();
+    }
+
+    public String getIpAddress()
+    {
+        return dockerContainer.getContainerIpAddress();
+    }
+
+    public int getPort()
```

Comment:
nit: consider `com.google.common.net.HostAndPort`:
```
return HostAndPort.fromParts(dockerContainer.getContainerIpAddress(), dockerContainer.getMappedPort(MONGO_PORT));
```

Reply:
or JDK's `InetSocketAddress` to avoid Guava ;)

Reply:
(we already use Guava in this module)